solutions/core-landing-zone/lz-folder/audits/logging-project/cloud-logging-buckets.yaml

```diff
@@ -19,7 +19,7 @@
 apiVersion: logging.cnrm.cloud.google.com/v1beta1
 kind: LoggingLogBucket
 metadata:
-  name: security-log-bucket # kpt-set: ${security-log-bucket}
+  name: security-log-bucket
```

Comment:
Does require a unique name to avoid naming collisions?

Reply:
Unlike storage buckets, logging buckets do not need to be globally unique. We can use the same bucket name in each of our dev LZs. What we need to make sure remains unique is the org sink names. In this case, we append the logging-project-id at the end: org-log-sink-security-${logging-project-id}

Reply:
Let me check with the team and see what they say. I just remember having issues using the same sink names for each of our dev LZs within the same organization.

Reply:
Did you mean collisions within GCP or perhaps config sync?

Reply:
Within a config controller namespace. Looking at it again, it's unlikely you'd be deploying 2 core LZ's within the same instance. I was also initially concerned about the uniqueness of the name but forgot that loggingbuckets didn't have the unique name requirements like buckets do.